packages/richtext-lexical/src/features/horizontalRule/client/nodes/HorizontalRuleNode.tsx

```diff
@@ -33,8 +27,8 @@ export class HorizontalRuleNode extends HorizontalRuleServerNode {
   /**
    * Allows you to render a React component within whatever createDOM returns.
    */
-  override decorate(): React.ReactElement {
-    return <HorizontalRuleComponent nodeKey={this.__key} />
+  override decorate() {
+    return null
```

Comment:
Why remove this?

Reply:
HorizontalRuleComponent doesn't do anything anymore. It used to be responsible for node selection and deletion, but that is now done in DecoratorPlugin.